x/leverage/keeper/collateral.go

```diff
@@ -1,8 +1,8 @@
 package keeper
 
 import (
+	sdkmath "cosmossdk.io/math"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-
```

Comment:
Our convention is to leave an empty line between cosmos sdk imports and umee repository imports

Reply:
Sure, will put that line back. It was my IDE optimizing imports